index.html

```diff
@@ -6,34 +6,43 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
   </head>
 
   <body>
     <header>
-      <div class="nav-bar">
-        <div class="nav-logo">
-          <a href="#">My Logo </a>
-        </div>
+      <div class="nav-logo">
+        <a href="#">My Logo </a>
+      </div>
 
-        <!---------- Desktop menu-------------->
+      <!---------- Desktop menu-------------->
 
-        <div class="desktop-menu">
-          <ul class="flex">
-            <li><a href="#">Portfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
-          </ul>
-        </div>
+      <nav class="desktop-menu">
+        <ul class="flex">
+          <li><a href="#cards" class="nav-list">Portfolio</a></li>
+          <li><a href="#about" class="nav-list">About</a></li>
+          <li><a href="#contact" class="nav-list">Contact</a></li>
+        </ul>
+      </nav>
 
-        <nav class="nav-links">
-          <div class="ambuger">
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
-          </div>
-        </nav>
+      <div class="nav-links">
+        <a href="#"
+          ><img
+            src="/images/icon-menu.png"
+            alt="navigation icon"
+            class="humberger"
+          />
+          <i class="fa-solid fa-xmark close"></i>
+        </a>
```

Comment:
- [ ] Good job adding the hamburger menu 👍🏽 , but I can see that your hamburger icon is not visible. Kindly fix the relative path of your hamburger icon to display the image

![image](https://user-images.githubusercontent.com/63473584/215584534-499a8342-bc68-4b6f-8d8d-5707b19da0f5.png)